src/adapter/compact.rs

```diff
@@ -0,0 +1,119 @@
+//! Module for use with `#[serde(with = "...")]` to serialize a [`Uuid`]
+//! as a `[u8; 16]
+//!
+//! [`Uuid`]: ../../struct.Uuid.html
+
+use serde::de::{self, Error, SeqAccess};
+use serde::ser::SerializeTuple;
+use serde::{Deserializer, Serializer};
+
+use prelude::*;
+
+use std::fmt;
+
+/// Serializer for a [`Uuid`] into a `[u8; 16]`
+///
+/// [`Uuid`]: ../../struct.Uuid.html
+pub fn serialize<S: Serializer>(
+    u: &Uuid,
+    serializer: S,
+) -> Result<S::Ok, S::Error> {
+    println!("sereeeeeeeeealize");
```

Comment:
uhmm?

Reply:
Debug messages, i'll take those out, whoops!

Reply:
wdym? Removed them in b1b2efe